docs/source/prototype.pipelines.rst

```diff
@@ -24,3 +24,29 @@ EMFORMER_RNNT_BASE_TEDLIUM3
 
    .. autodata:: EMFORMER_RNNT_BASE_TEDLIUM3
       :no-value:
+
+
+HiFiGAN Generator
+-----------------
+
+Interface
+~~~~~~~~~
+
+``HiFiGANGeneratorBundle`` defines HiFiGAN Generator pipeline capable of transforming mel spectrograms into waveforms.
+
+.. autosummary::
+   :toctree: generated
+   :nosignatures:
+   :template: autosummary/bundle_class.rst
+
+   HiFiGANGeneratorBundle
+
+.. minigallery:: torchaudio.prototype.pipelines.HiFiGANGeneratorBundle
```

Comment:
Since there is no tutorial using this bundle, this line can be removed.

Reply:
Done